tools/topology/topology2/platform/intel/speaker-echo-ref.conf

```diff
@@ -15,7 +15,7 @@ Object.Pipeline {
 		index	$ECHO_REF_DAI_PIPELINE_ID
 
 		Object.Widget.copier."1" {
-			dai_index	1
+			dai_index	$SPEAKER_SSP_DAI_INDEX
 			dai_type	"SSP"
 			type		dai_out
```

Comment:
@ranj063 out of curiosity is there a reason why we never use dai_in? It's implied or the default?

Reply:
possibly dai_in enumerates to 0 and becomes default. @ranj063 do we need a followup for the copier base to make this a mandatory setting ?